.codeclimate.yml

```diff
@@ -0,0 +1,4 @@
+exclude_patterns:
```

Comment:
Hello @davenjeru ,
Here is how I ignored migrations and tests from code climate analysis